resources/Data.sql

```diff
@@ -12,7 +12,7 @@ create table ticket(
  ID int PRIMARY KEY AUTO_INCREMENT,
  PARKING_NUMBER int NOT NULL,
  VEHICLE_REG_NUMBER varchar(10) NOT NULL,
- PRICE double,
+ PRICE varchar(10),
```

Comment:
Attention à la taille. Ici tu as mis 10 caractères or un prix peut avoir plus 64 caractères. Je mettrai bien 
```suggestion
 PRICE varchar(64),
```